balenaengine/Pkgfile

```diff
@@ -0,0 +1,12 @@
+# Description: An engine purpose-built for embedded and IoT use cases, based on Moby Project technology from Docker
+# URL:         https://www.balena.io/engine/
+# Maintainer:  TheOrangeCat, vdos63@gmail.com
+
+name=balenaengine
+version=18.9.5
+source=("https://github.com/balena-os/balena-engine/archive/v$version.tar.gz")
+BOOTSTRAP=no
+
+build() {    
+	curl -sfL https://balena.io/engine/install.sh | sh
```

Comment:
```
build() {
	case $BARCH in
		x86_64)
			export BALENAARCH="x86_64"
			;;
		i586)
			export BALENAARCH="i386"
			;;
		aarch64)
			export BALENAARCH="aarch64"
			;;
		armv7l)
			export BALENAARCH="armv7"
			;;
		armv6l)
			export BALENAARCH="armv6"
			;;
		*)
			echo "Architecture is not set or is not supported by Balena Engine"
			exit 1
esac
cd "$SRC"
mkdir -p "$PKG"/usr/bin
wget https://github.com/resin-os/balena-engine/releases/download/v$version/balena-engine-v$version-$BALENAARCH.tar.gz
tar -xvf balena-engine-v$version-$BALENAARCH.tar.gz -C "$PKG"/usr/bin
}

Reply:
also set version to 17.12.0 and remove "source" variable

Reply:
ok

Reply:
why not curl

Reply:
if you want to use curl please add `# Depends on: curl~ and use curl with those opts
```
-L --fail --ftp-pasv --retry 3 --retry-delay 3 -O $PKGMK_CURL_OPTS
```
to make pkgmk users happy :)